src/modules/coin/mod.rs

```diff
@@ -4,29 +4,36 @@ mod data_provider;
 mod resources;
 mod schemas;
 
+#[derive(serde::Serialize)]
+pub struct ValidationErrorJsonPayload {
+    pub message: String,
+    pub fields: Vec<String>,
+}
+
```

Comment:
Nit
We can drop this. Also, please run `cargo fmt`, there should be no changes in this file and in `nft/mod.rs`